pipeline.yml

```diff
@@ -37,7 +37,7 @@ resources:
   type: git
   source:
     uri: https://github.com/18F/cg-compliance.git
-    branch: master
+    branch: update2
```

Comment:
I'm assuming this is meant to be temporary? If you find yourself needing to change this often, I would change it to be a template variable instead.

http://concourse.ci/fly-set-pipeline.html